tsconfig.json

```diff
@@ -25,7 +25,9 @@
       "components/*": ["src/components/*"],
       "pages/*": ["src/pages/*"],
       "routes/*": ["src/routes/*"],
-      "test/*": ["src/test/*"]
+      "test/*": ["src/test/*"],
+      "utils/*": ["src/utils/*"],
+      "types/*": ["src/types/*"]
     },
     "types": ["vitest/globals"]
```

Comment:
ja osobisćie nie jestem fanem używania globalsów, zawsze są z nimi jakieś problemy 😭

tbh lepiej jest po prostu importować te funkcje

ale to nie jest jakiś błąd, moja preferencja